updates/next.json

```diff
@@ -1,16 +1,24 @@
 {
   "stream": "next",
   "metadata": {
-    "last-modified": "2020-06-22T18:38:34Z"
+    "last-modified": "2020-06-25T15:18:18Z"
   },
   "releases": [
     {
-      "version": "32.20200615.1.2",
+      "version": "32.20200615.1.3",
+      "metadata": {
+        "barrier": {
+          "reason": "https://github.com/coreos/fedora-coreos-tracker/issues/484"
+        }
+      }
+    },
+    {
+      "version": "32.20200615.1.3",
       "metadata": {
         "rollout": {
-          "start_epoch": 1592859600,
+          "start_epoch": 1593100800,
           "start_percentage": 0.0,
-          "duration_minutes": 2880
+          "duration_minutes": 1440
```

Comment:
should we make this less  than a day so we can start the next rollout tomorrow morning (US eastern time)?

Reply:
Hmm, yeah sounds good to me.